source/reference/compatibility.txt

```diff
@@ -112,6 +112,14 @@ specified for the operation:
 In the legacy ``mongo`` shell, these operations use the specified read
 preference.
 
+Write Preference Behavior
+-------------------------
+
+:ref:`Retryable writes <retryable-writes>` are the default in
```

Comment:
How about this to be a bit more transparent that the default is `on/true`
```suggestion
:ref:`Retryable writes <retryable-writes>` are enabled by default in
```

Reply:
Done